src/widgets/events/ui/events.module.scss

```diff
@@ -24,6 +24,14 @@
       align-items: center;
       justify-content: space-between;
 
+      > img {
```

Comment:
Maybe we could use a more unique class selector here than the image tag?

Reply:
Done